lib/puppet-strings/yard/handlers/helpers.rb

```diff
@@ -0,0 +1,8 @@
+module PuppetStrings::Yard::Handlers::Helpers
+  # Logs a warning if a summary tag has more than 140 characters
+  def self.validate_summary_tag(object)
+    if object.has_tag?(:summary) && object.tag(:summary).text.length > 140
```

Comment:
@scotje I stuck this in this Helper module for the Handlers. Does that seem reasonable?